### PR TITLE
Add scikit issue to known bugs docs

### PR DESCRIPTION
**Pull request by javerbukh**
Document the bug mentioned in #490

### CODE REVIEW

File: docs/known_bugs.rst [hunk 1]
```diff
@@ -1,6 +1,14 @@
 Known Issues
 ============
 
+On some platforms, install fails due to `scikit-image`
```
Comment (rosteen) [inline docs/known_bugs.rst:4]: Would it be useful to note that this is only applicable to users working in a conda environment? It seems obvious, but... *shrug*

**Review by rosteen — APPROVED** [review]

File: docs/known_bugs.rst (hunk 1 above)
Comment (javerbukh) [reply docs/known_bugs.rst]: Good call!

File: docs/known_bugs.rst [hunk 2]
```diff
@@ -1,6 +1,14 @@
 Known Issues
 ============
 
+In some conda environments, install fails due to `scikit-image`
+---------------------------------------------------------------
+
+This can be fixed by running::
```
Comment (duytnguyendtn) [inline docs/known_bugs.rst:7]:
```suggestion
This can be fixed by reinstalling `scikit-image`::
```

**Review by duytnguyendtn — CHANGES_REQUESTED** [review]
Super small change to clarify what these lines of code are doing

**Comment by javerbukh** [comment]
@rosteen @duytnguyendtn Definitely a good call on clarifying the problem in the documentation, thank you both!

**Review by rosteen — APPROVED** [review]
LGTM

**Pull request merged**